std_msgs/String task
std_msgs/String req
---
zeabus_vision/vision_buy_a_gold_chip data
